docusaurus/docs-internals/editor-controls/multivalue-controls/motion.md

```diff
@@ -0,0 +1,190 @@
+| `helper?.content`  | `string`                                                                                                                                                                                   |    -    | If provided, displays an icon next to the label. When hovering over this icon, a tooltip with additional information appears                                                                                                                                       |
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon                                                                                                                                                                                                  |
+| `config?.disabled` | `Array<Effect>`                                                                                                                                                                            |  `[ ]`  | An array of animation names that should be disabled. They will not disappear from the list, but they cannot be selected <br/> <br/> <b>`Effect = "vertical" \| "horizontal" \| "transparency" \| "blur" \| "rotate" \| "scale" \| "mouseTrack" \| "mouseTilt"`</b> |
+
+### Return value
+
+Returns an object with the following values:
+
+```js
+{
+  active: undefined | "vertical" | "horizontal" | "transparency" | "blur" | "rotate" | "scale" | "mouseTrack" | "mouseTilt";
+  vertical: undefined | { speed: number; direction: "up" | "down"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  horizontal: undefined | { speed: number; direction: "left" | "right"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  transparency: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  blur: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  rotate: undefined | { direction: "left" | "right"; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; speed: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  scale: undefined | { direction: "up" | "down" | "downUp" | "upDown"; speed: number; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  mouseTrack: undefined | { direction: "direct" | "opposite"; speed: number; };
+  mouseTilt: undefined | { direction: "direct" | "opposite"; speed: number; };
+}
+```
+
+All the object values are the animation names which contains general parameters:
```

Comment:
need to explain what `active` or other non general parameter do, then need to explain what general parameters does ( as it is at the moment )